Assets/MixedRealityToolkit.Providers/OpenVR/GenericOpenVRController.cs

```diff
@@ -82,6 +82,7 @@ public GenericOpenVRController(TrackingState trackingState, Handedness controlle
             new MixedRealityInteractionMapping(9, "Unity Button Id 3", AxisType.Digital, DeviceInputType.ButtonPress, KeyCode.JoystickButton3),
             new MixedRealityInteractionMapping(10, "WMR Touchpad Touch", AxisType.Digital, DeviceInputType.TouchpadTouch, KeyCode.JoystickButton18),
             new MixedRealityInteractionMapping(11, "WMR Touchpad Position", AxisType.DualAxis, DeviceInputType.Touchpad, ControllerMappingLibrary.AXIS_17, ControllerMappingLibrary.AXIS_18),
+            new MixedRealityInteractionMapping(12, "WMR Grip Pose", AxisType.SixDof, DeviceInputType.SpatialGrip, MixedRealityInputAction.None),
```

Comment:
Is this specific to WMR? I believe this should work regardless of controller / platform: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/f3a97916629a965c9fc45e85290c453ebc9b45f3/Assets/MixedRealityToolkit/Providers/UnityInput/GenericJoystickController.cs#L207

Reply:
Arguably, it's a mistake when I first put that forward as we only had WMR to test on.  These should all be called "Generic" as they are per the unaltered generic spec, not specific to any device.